src/main/java/io/cdap/plugin/SCPRemotetoRemoteAction.java

```diff
@@ -0,0 +1,293 @@
+package io.cdap.plugin;
+
+/*
+ * Copyright © 2017 Cask Data, Inc.
```

Comment:
Wrong year. Also, the copyright text should be before any code. Move it before the `package` line.